.github/workflows/api.test.yml

```diff
@@ -9,12 +9,18 @@ jobs:
       run:
         working-directory: ./bctw-api
     steps:
-      - uses: actions/checkout@v1
-      - uses: actions/setup-node@v3
+      - name: Checkout Repo
+        uses: actions/checkout@v1
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
         with:
           node-version: "14"
           cache: "npm"
-      - name: Setup Node
+          cache-dependency-path: bctw-api/package-lock.json
```

Comment:
I deleted an empty package-lock.json at the root level, which made this step fail. Turns out the working-directory setting above only applies to 'runs' and not 'with'. So this fixes the issue that arose when I removed that random lock file.